ruby/rails/security/brakeman/check-validation-regex.yaml

```diff
@@ -0,0 +1,33 @@
+rules:
+- id: check-validation-regex
+  mode: search
+  patterns:
+  - pattern-either:
+    - pattern: |
+        validates ..., :format => <... $V ...>,...
+    - pattern: |
+        validates_format_of ..., :with => <... $V ...>,...
+  - metavariable-regex:
+      metavariable: $V
+      regex: /(.{2}(?<!\\A)[^\/]+|[^\/]+(?<!\\[Zz]))\/
+  message: $V Found an incorrectly-bounded regex passed to `validates_format_of` or
```

Comment:
I noticed at https://brakemanscanner.org/docs/warning_types/format_validation/ it mentions that a lowercase `z` should be used. Does capital `Z` perform a similar function?

Reply:
it does, or Brakeman itself has a bug: https://github.com/presidentbeef/brakeman/blob/main/lib/brakeman/checks/check_validation_regex.rb#L67

I used the underlying Brakeman implementations to write all of my ports